### ISSUE TITLE
Attempting to replicate results

### ISSUE THREAD

**Issue by patleitao** [trypag/torchmed#4]
Hello @trypag,

Thank you for releasing the code to your great paper! 

I am currently trying to integrate your NonAdj loss to a current project, and have been attempting to replicate your results, using the code you provide in your second example. I faced quite a few problems and was wondering if you would have any insights.

First, I am a bit confused with the way you sample inputs for your model. It seems that you sample 184x184 patches, which produce cropped slices. I was wondering why you don't use the 256x256 original size and how you integrate your patch results to obtain a full brain segmentation.

Additionally, when I run the file segment.py with the pre-trained weights that you provide, the code either throws an error or doesn't finish (after I comment out the lines giving the error), as it seems it gets stuck in the sampling loop. 

I have also trained a new model with the existing code, but have been struggling to visualize results with the given model, given the problems mentioned above. 

Any help would be greatly appreciated!

Thank you

**Comment by trypag** [comment]
Hello ,
> First, I am a bit confused with the way you sample inputs for your model. It seems that you sample 184x184 patches, which produce cropped slices. I was wondering why you don't use the 256x256 original size and how you integrate your patch results to obtain a full brain segmentation.

All of my images where re-sampled to size 184x184x216 (or something like this), during the registration step. Thus, I don't use patches as inputs but complete slices.

> Additionally, when I run the file segment.py with the pre-trained weights that you provide, the code either throws an error or doesn't finish (after I comment out the lines giving the error), as it seems it gets stuck in the sampling loop.

There might be an error with your sampling strategy, or it can be related to your data. I would need a reproduction to give some feedback.

**Comment by patleitao** [comment]
Thank you for the reply! Yes, it seems my dataset wasn't preprocessed in the same way. By passing in my data to your code, it actually samples in axial plan, instead of coronal.

So to make sure, if I take the original MICCAI dataset and register it using FSL FLIRT do I end up with 184x184x216?

**Comment by trypag** [comment]
If you use the same preprocessing as [mine](https://github.com/trypag/torchmed/blob/master/examples/scripts/preprocessing/miccai12.py), it should be fine.

You could also adapt the code to your image size, but before that make sure all of your image have the same size and orientation, that's why we register in the first place.
